runelite-client/src/main/java/net/runelite/client/plugins/microbot/runecrafting/gotr/GotrScript.java

```diff
@@ -150,13 +149,12 @@ public boolean run(GotrConfig config) {
                     if (repairCells()) return;
 
                     if (usePortal()) return;
-                    //mine huge guardian remains
```

Comment:
Why remove informational comments that the original developer placed here?

Reply:
Refactor

Reply:
I dont see any benefit of doing this, just make things more ambiguous